dm/loader/checkpoint.go

```diff
@@ -470,19 +466,44 @@ func (cp *RemoteCheckPoint) String() string {
 	return string(bytes)
 }
 
+type lightingLoadStatus int
+
+const (
+	lightningStatusInit lightingLoadStatus = iota
+	lightningStatusRunning
```

Comment:
maybe reuse `CheckpointStatus` in `tidb/pkg/checkpoints` ?

Reply:
do you mean the real tidb-lightning's `CheckpointStatus` ? I think because there are so many state in it, it maybe hard to understand of not familiar with tidb-lightning

Reply:
done